plugin/github.com.hashicorp.go.kms.wrapping.plugin.v2.proto

```diff
@@ -82,5 +82,21 @@ message HmacKeyIdResponse {
 
 message KeyBytesRequest {};
 message KeyBytesResponse {
-	bytes key_bytes = 10 [json_name="key_bytes"];
+	bytes key_bytes = 10;
+}
+
+message SignRequest {
+	github.com.hashicorp.go.kms.wrapping.v2.types.Options options = 10;
+	bytes msg = 20;
```

Comment:
What is the format of this message? Encoded protobuf?

Reply:
does it matter?  we're signing the bytes of whatever.

Reply:
Ah, in that case I'd prefer just to call this "payload". Makes it a bit clearer to me that this is just arbitrary bytes to be signed.

Reply:
typical nomenclature is to "sign a message"